packages/devextreme/scss/widgets/base/_form.scss

```diff
@@ -123,6 +123,12 @@
   font-size: 20px;
 }
 
+.dx-form-group-custom-caption {
+  .dx-icon {
+    vertical-align: top;
```

Comment:
Why 'top' value is used?

Reply:
it's the same value as for dx-icon in custom label. without it icon position looks bad